File: PokerProtocol/src/protocol/game/commands/TableClosedCommand.java
package protocol.game.commands;

import java.util.StringTokenizer;

import protocol.ICommand;

public class TableClosedCommand implements ICommand
{
    public static String COMMAND_NAME = "gameTABLE_CLOSED";
    
    public TableClosedCommand(StringTokenizer argsToken)
    {
    }
    
    public TableClosedCommand()
    {
    }
    
    @Override
    public String encodeCommand()
    {
        final StringBuilder sb = new StringBuilder();
        sb.append(TableClosedCommand.COMMAND_NAME);
        return sb.toString();
    }
}

File: trunk/C#/PokerProtocol/Commands/Game/TableClosedCommand.cs
using System;
using System.Collections.Generic;
using System.Text;
using EricUtility;
using EricUtility.Networking.Commands;

namespace PokerProtocol.Commands.Game
{
    public class TableClosedCommand : AbstractJsonCommand
    {
        public static string COMMAND_NAME = "gameTABLE_CLOSED";

        public TableClosedCommand()
        {
        }
    }
}
